Makefile

```diff
@@ -19,7 +19,7 @@ CGO_ENABLED=1
 export CGO_ENABLED
 
 # Get rid of useless warning in lmdb
```

Comment:
Please update the comment too

Reply:
The warning is still in `lmdb`, so it's still right, but I can remove it still if you like

Reply:
No, if that is the case it is fine. I thought it came from somewhere else.